cli.sh

```diff
@@ -157,10 +158,10 @@ find_hook_path_to_enable_or_disable() {
 
             REMOTE_URL=$(git -C "$SHARED_ROOT" config --get remote.origin.url)
 
-            SHARED_LOCAL_REPOS_LIST=$(grep -E "^[^#].+$" <"$(pwd)/.githooks/.shared")
-            ACTIVE_LOCAL_REPO=$(echo "$SHARED_LOCAL_REPOS_LIST" | grep -o "$REMOTE_URL")
+            SHARED_LOCAL_REPOS_LIST=$(grep -E "^[^#\n\r ].*$" <"$(pwd)/.githooks/.shared")
+            ACTIVE_LOCAL_REPO=$(echo "$SHARED_LOCAL_REPOS_LIST" | grep -F -o "$REMOTE_URL")
 
-            ACTIVE_GLOBAL_REPO=$(git config --global --get githooks.shared | grep -o "$REMOTE_URL")
+            ACTIVE_GLOBAL_REPO=$(git config --global --get-all githooks.shared | grep -o "$REMOTE_URL")
```

Comment:
Is this change backwards compatible?
Will we still correctly process existing settings with comma-separated repository lists?

Reply:
No, not backward compatible. As soon as you update, everything gets transformed -> see test119.sh